File: programme/swap-two-numbers/swap.js
// Write a programmeto swap two numbers

"use strict";

// perform following command to run programme
// node swap.js

const readline = require("readline");

const rl = readline.createInterface({
  input: process.stdin,
  output: process.stdout,
});

rl.question("Enter a : ", (a) => {
  rl.question("Enter b : ", (b) => {
    [a, b] = [a, b].map(Number);
    if (isNaN(a) || isNaN(b)) {
      console.log("Some error occured");
    } else {
      let tmp = b;
      b = a;
      a = tmp;
      console.log(`\nOutput:\na = ${a}\nb = ${b}`);
    }
    rl.close();
  });
});

File: programme/swap-two-numbers/Swap.java
// Write a programmeto swap two numbers
import java.util.Scanner;

public class Swap {

  public static void main(String[] args) {
    Scanner sc = new Scanner(System.in);
    int a, b;

    System.out.print("Enter a : ");
    a = sc.nextInt();
    System.out.print("Enter b : ");
    b = sc.nextInt();

    int c = a;
    a = b;
    b = c;
    System.out.println("\nOutput after swapping :\na : " + a + "\nb : " + b);

    sc.close();
  }
}
